azkaban-common/src/main/java/azkaban/executor/ExecutionLogsDao.java

```diff
@@ -67,10 +67,29 @@ LogData fetchLogs(final int execId, final String name, final int attempt,
     }
   }
 
+  public void appendLogs(final int execId, final String name, final int attempt, final File... files)
```

Comment:
Can you add javadoc style comment on top of the method?

Reply:
Added, thank you!